app/components/primer/border_box_component.rb

```diff
@@ -13,6 +13,14 @@ class BorderBoxComponent < Primer::Component
     }.freeze
     PADDING_SUGGESTION = "Perhaps you could consider using :padding options of #{PADDING_MAPPINGS.keys.to_sentence}?"
 
+    DEFAULT_ROW_SCHEME = :default
+    ROW_SCHEME_MAPPINGS = {
+      DEFAULT_ROW_SCHEME => "",
+      :yellow => "Box-row--yellow",
+      :blue => "Box-row--blue",
+      :gray => "Box-row--gray"
```

Comment:
In color modes v2 work, these names will be changed, wonder if we should use new names now since this will need to be changed? [ref](https://docs.google.com/spreadsheets/d/1vL-FPM7-NvTsHgKBRur543jnuzQJqJ69Fl6pH8EyKy4/edit#gid=752100387)

```suggestion
      :warning => "Box-row--yellow",
      :info => "Box-row--blue",
      :neutral => "Box-row--gray"
```

Reply:
I like that. One less breaking API change!